File: AngularClient/src/app/_models/message.ts
export interface Messages {
  id: number;
  senderId: number;
  senderUsername: string;
  senderPhotoUrl: string;
  recipeintId: number;
  recepeintUsername: string;
  recipeintPhotoUrl: string;
  content: string;
  dateRead: Date;
  messageSent: Date;
}

File: Data/Entities/Message.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Data.Entities
{
    public class Message
    {
        public int Id { get; set; }
        public int SenderId { get; set; }
        public string SenderUsername { get; set; }
        public ApplicationUser Sender { get; set; }
        public int RecipeintId { get; set; }
        public string RecepeintUsername { get; set; }
        public ApplicationUser Recipeint { get; set; }
        public string Content { get; set; }
        public DateTime? DateRead { get; set; }
        public DateTime MessageSent { get; set; } = DateTime.UtcNow;
        public bool SenderDeleted { get; set; }
        public bool RecipeintDeleted { get; set; }
    }
}
